msteams-platform/task-modules-and-cards/what-are-cards.md

```diff
@@ -68,6 +68,26 @@ Type ahead search added as an input control in Adaptive Cards enable [dynamic se
 
 Adaptive Cards with Incoming Webhooks enables you to use the rich and flexible capabilities of Adaptive Cards. It sends data using Incoming Webhooks in Teams from their web service.
 
+### Adaptive cards form completion
+
+**Adaptive cards form completion** is a visual informational feedback message that appears when card action completes to a configurable set of card actions.
+Adaptive cards extensions are impacted with this feature
+
+User can see two types of feedback messages.
+
+**Error:** When a card action completes, user receives feedback as **Something went wrong, Try again** message if there is an error in the attempt.
```

Comment:
Error: If there is an error while sending a response to bot, Something went wrong, Try again" message appears.